builder/bundles.go

```diff
@@ -877,7 +878,40 @@ func createVersionsFile(baseDir string, packagerCmd []string) error {
 	return w.Flush()
 }
 
-func fixOSRelease(filename, version string) error {
+func fixOSRelease(b *Builder, filename, version string) error {
+
+	//Replace the default os-release file if customized os-release file found
+	path := b.Config.Mixer.OSReleasePath
+	if path != "" {
+
+		// Validate file path
+		_, err := os.Stat(path)
+		if err != nil {
+			return err
+		}
+
+		fSrc, err := os.OpenFile(path, os.O_RDONLY, 0666)
+		if err != nil {
+			return err
+		}
+
+		fDst, err := os.OpenFile(filename, os.O_RDWR, 0666)
+		if err != nil {
+			return err
+		}
+
+		//Overwrite the os-release file
+		_, err = io.Copy(fDst, fSrc)
+		if err != nil {
+			return err
+		}
+
+		//Close file descriptor to make sure the copy operation is completed.
+		_ = fSrc.Close()
+		_ = fDst.Close()
+
+	}
+
```

Comment:
Even thought the code above looks good, we could improve it a bit. In the code bellow we open os-release file to replace the version number. To do so, we read all the content and rewrite the entire file at the end. As such, instead of replacing the file with the custom one, we can just change where we read the content from. 
If we do so, the whole code above can be removed by replacing the line below with:

```
var err error
var f *os.File
if b.Config.Mixer.OSReleasePath != "" {
    f, err = os.Open(b.Config.Mixer.OSReleasePath)
} else {
    f, err = os.Open(filename)
}
```

Reply:
Good suggestion and I like this.

Reply:
@rchiossi , Thanks for your review and I had push new change.